src/web/lib/hakuneko/frontend@classic-dark/menu.html

```diff
@@ -142,6 +147,13 @@
                 <div class="logo">
                     <img src="./img/logo_m.png"/>
                 </div>
+                <div class="contact">
+                    <div >
+                        <img src="http://icons.iconarchive.com/icons/papirus-team/papirus-apps/48/discord-icon.png"/>
```

Comment:
Images should be placed in the `img` sub-directory so that they are available when HakuNeko is offline